explorer/client/src/components/Cards/Title.tsx

```diff
@@ -0,0 +1,23 @@
+import CardContent from '@material-ui/core/CardContent'
```

Comment:
Had to manually copy this component over from the storybook. Can remove once...
https://www.pivotaltracker.com/story/show/167792209
...is completed

Reply:
Lets add capitalized FIXMES/REMOVEMES lest it is forgotten